torchci/components/job/GroupJobConclusion.tsx

```diff
@@ -430,26 +489,42 @@ function ToolTip({
   groupName,
   message,
   jobs,
+  rowData,
 }: {
   conclusion: string;
   groupName: string;
   message: string;
   jobs: JobData[];
+  rowData?: RowData;
 }) {
+  // Import isJobAutorevertSignal at the top of this component if needed
+  const { isJobAutorevertSignal } = require("lib/autorevertUtils");
+
   return (
     <div>
       {`[${conclusion}] ${groupName}`}
       <div>{message}</div>
       {jobs.map((job, ind) => {
+        const isAutorevert = rowData
+          ? isJobAutorevertSignal(job, rowData)
+          : false;
         return (
           <a
             key={ind}
             href={job.htmlUrl}
             target="_blank"
             rel="noreferrer"
-            style={{ display: "block" }}
+            style={{
+              display: "block",
+              fontWeight: isAutorevert ? "bold" : "normal",
+              color: isAutorevert ? "#d73a49" : undefined,
```

Comment:
yeah, I think it makes sense to make it a css class, because currently the style is duplicating in two places